source/backend/plugin/CarlaPluginFluidSynth.cpp

```diff
@@ -73,20 +73,22 @@ class CarlaPluginFluidSynth : public CarlaPlugin
         fSynth = new_fluid_synth(fSettings);
         CARLA_SAFE_ASSERT_RETURN(fSynth != nullptr,);
 
+        keepFluidDefaults();
```

Comment:
I dont like some the naming here. please rename to `InitializeFluidDefaultsIfNeeded` (which I believe is a lot more clear on what it does.

Reply:
err, lowercase I, so `initializeFluidDefaultsIfNeeded`